### ISSUE TITLE
How do I install RDataTracker?

### ISSUE THREAD

**Issue by dlebauer** [blernermhc/RDataTracker#60]
I've tried

```r
install.packages("RDataTracker")
```
> Warning in install.packages :
  package ‘RDataTracker’ is not available (for R version 3.1.0)

and 

```r
devtools::install_github("blernermhc/RDataTracker")
```
> Error in loadNamespace(package, c(which.lib.loc, lib.loc)) : 
  cyclic namespace dependency detected when loading ‘RDataTracker’, already loading ‘RDataTracker’
Error : package or namespace load failed for ‘RDataTracker’
Error : unable to load R code in package ‘RDataTracker’
ERROR: lazy loading failed for package ‘RDataTracker’

Platforms: same results on RedHat 5 / R 3.1.0 and Ubuntu 14.04 / R 3.1.3

**Comment by dlebauer** [comment]
The errors above occur even after restarting R in 'vanilla' mode and checking that there is no existing installation of RDataTracker. 

The version at the Harvard archive works: 

```sh
wget http://harvardforest.fas.harvard.edu/data/p09/hf091/RDataTracker_1.000.tar.gz", destfile = "RDataTracker_1.000.tar.gz
R CMD INSTALL RDataTracker_1.000.tar.gz
```

**Comment by blernermhc** [comment]
Just to clarify...  Do you have RDataTracker successfully installed now?

RDataTracker is not available via CRAN, so that is what caused the first error.  We will need to look into the install_github problem.  Thanks for reporting it.

**Comment by dlebauer** [comment]
Yes, installing from the archived binary worked. Thanks,

David

**Comment by dlebauer** [comment]
Should be fixed by #61

**Comment by dlebauer** [comment]
sorry. this shouldn't be closed until #61 is merged ...

**Comment by blernermhc** [comment]
Merged in the change from #61
